cfgov/jinja2/v1/ask-cfpb/answer-page.html

```diff
@@ -34,22 +34,42 @@ <h3 class="answer-snippet">
                 {{ page.snippet | safe }}
             </h3>
         {% endif %}
-        <div>
-            {{ page.answer | safe }}
-
-        </div>
-
-        {% if audiences %} 
-            <div class="u-mt15">
-                {{ tags.render( {'links': audiences}, '', is_wagtail=True) }}
+        <div data-qa-hook="expandable" 
+             data-read-more="true"
+             class="o-expandable
+                    o-expandable__read-more">
+            <div class="o-expandable_content" id="answer-content">
+                <div class="o-expandable_content-animated">
+                    <div class="answer-text">
+                        {{ page.answer | safe }}
+                    </div>
+                    {% if audiences %} 
+                    <div class="u-mt15">
+                        {{ tags.render( {'links': audiences}, '', is_wagtail=True) }}
+                    </div>
+                    {% endif %}
+
+                    {# This brings in the feedback module #}
+                    {% for block in page.content %}
+                        {{- form_block.render(block, 'content', loop.index0) -}}
+                    {% endfor %}
+                </div>
+                <div class="o-expandable__read-more_gradient"></div>
```

Comment:
Could the gradient be on the `o-expandable_content` element using a pseudo element?